tests/compute/subresources/vmi.go

```diff
@@ -122,7 +123,7 @@ var _ = compute.SIGDescribe("VirtualMachineInstance subresource", func() {
 			}, 30*time.Second, 2*time.Second).Should(BeTrue())
 		}
 
-		It("[test_id:7479]Freeze Unfreeze should succeed", func() {
+		It("[test_id:7479]Freeze Unfreeze should succeed", decorators.Conformance, func() {
```

Comment:
I would hold these tests to higher standard. I would like to see the setup here.